metrics/proc-metrics.go

```diff
@@ -21,23 +21,17 @@ func SetupMetrics(port int) error {
 	procCollector := prom.NewProcessCollector(prom.ProcessCollectorOpts{})
 	registry.MustRegister(goCollector, procCollector)
 	pe, err := prometheus.NewExporter(prometheus.Options{
-		Namespace: "hydra-booster",
+		Namespace: "hydrabooster",
 		Registry:  registry,
 	})
 	if err != nil {
-		return err
+		log.Fatalf("Failed to create exporter: %v", err)
 	}
 
-	_ = view.RegisterExporter
-	// NOTE: Disabling opencensus for now, it allocates too much
```

Comment:
🚩 this might still be a problem for running >75 sybils on a single machine.

Reply:
We might be able to get more just by using GKE